lib/psql.js

```diff
@@ -0,0 +1,40 @@
+'use strict'
+
+const co = require('co')
+const debug = require('debug')('psql')
+
+function * exec (db, query) {
+  const execa = require('execa')
+  const os = require('os')
+  const cli = require('heroku-cli-util')
+
+  debug(query)
+  let env = {
+    PGAPPNAME: 'psql non-interactive',
```

Comment:
So stupid question but is this only for the non-interactive commands? How does `pg:psql` work now?